changelog.txt

```diff
@@ -6,4 +6,5 @@ fixed - Fixed an error that occurs when a Firestore field override removes all i
 feature - Firestore emulator now has the ability to produce rule-coverage reports
 changed - Firestore emulator now exposes the v1 service definition
 changed - Firestore emulator has various runtime improvements
-changed - Clearer empty state when pretty-printing Firestore indexes
\ No newline at end of file
+changed - Clearer empty state when pretty-printing Firestore indexes
+fixed - Updated CLI update command to show global (`-g`) flag
```

Comment:
Honestly I'm not sure this is a change that merits a changelog entry since it doesn't affect user behavior.

Reply:
Okay.